mne/io/artemis123/utils.py

```diff
@@ -82,3 +84,55 @@ def _compute_mne_loc(coil_loc):
     R = rotation3d_align_z_axis(z_axis)
     loc[3:13] = R.T.reshape(9)
     return loc
+
+
+def _read_pos(fname):
```

Comment:
You are correct that both of these methods read in files generated by the same digitization program (written by CTF). The issue is that I don't think the behavior of `_read_pos` in `mne/io/ctf/eeg.py` is correct.

For instance the transform used to move the digitization points to head space is computed using the fid locations computed for the MEG HPIs not their polhemous coordinates. Also the coord_frame of the locations (`FIFFV_MNE_COORD_CTF_HEAD`) isn't correct once the transformation `transformations['t_ctf_head_head']` is applied.

That stuff I could make backward compatible but the ident coding (https://github.com/mne-tools/mne-python/blob/master/mne/io/ctf/eeg.py#L83-L87) is harder to maintain and also doesn't make a whole lot of sense.

Assuming there Is there still a refactoring of the dig structure planned, my vote would be to leave the code as is and to add both methods to the list of things that need to be rethought and refactored. as part of that refactoring effort.